pyperf/_metadata.py

```diff
@@ -42,6 +42,12 @@ def is_positive(value):
     return (value >= 0)
 
 
+def is_tags(value):
+    if not isinstance(value, (list, tuple)):
+        return False
+    return all(isinstance(x, str) and x != 'all' for x in value)
```

Comment:
I suggest to reject empty tags. They would be annoying for users, like specifying an empty tag on the command line.